aries-site/src/examples/templates/forms/PayExample.js

```diff
@@ -118,6 +140,9 @@ export const PayExample = () => {
           pad={{ horizontal: 'xxsmall' }}
         >
           <Form
+            messages={{
+              required: '! This is required',
```

Comment:
Would we consider updating grommet to allow a node as required message type? Designs call for the `!` to be bold but know this would require doing something with `<Box />` potentially.

Reply:
yes I left it for now just until that icon is made and then I can have an issue to refer to for grommet and then fix prop types in grommet